meet-data/rpu/1703/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+WRPF,2017-04-08,Russia,,Volzhsky,Территориальный чемпионат Юга России
```

Comment:
WRPF?

The MeetName is "South Russia Championship", litreal translation, "territorial championship of the south of Russia"